gst/nnstreamer/tensor_crop/tensor_crop.c

```diff
@@ -0,0 +1,208 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2021 Samsung Electronics Co., Ltd.
+ */
+
+/**
+ * SECTION:element-tensor_crop
+ *
+ * GStreamer element to crop the regions of incoming tensor.
```

Comment:
Please start the documentation section (Line 7-9) after Doxygen section. Please add an pipeline example and some descriptions as well.

Reply:
updated, I added todo tag for later work - element description and example pipeline.

Reply:
updated, I added todo tag for later work - element description and example pipeline.
After adding new PRs, I will update this section.
- define struct GstTensorCropInfo (tensor_typedef.h)
- support 2 sink pads (raw + info) in tensor_crop
- note (layout only NHWC supported)